packages/twenty-front/src/modules/workflow/components/FormFieldInput.tsx

```diff
@@ -0,0 +1,26 @@
+import VariableTagInput from '@/workflow/search-variables/components/VariableTagInput';
```

Comment:
we need that component and the variables handling to be outside of workflows. But not for this PR!